### DISCUSSION TITLE
Erro no Portal do PO-UI

### DISCUSSION THREAD

**Discussion by JCTG-Jorge** [po-ui/po-angular#2042] [Angular] [answered]
Boa tarde!

Está dando erro ao acessar o portal do po-ui...está tendo manutenção? 
![image](https://github.com/po-ui/po-angular/assets/160171828/0d23d137-559d-47c6-9125-75693f7526b1)

**Comment by anliben** [comment, marked as answer]
Bom Dia. Em breve expediremos um novo ssl do nosso portal, e todas a API estará funcionando.
